lib/vm/src/vmcontext.rs

```diff
@@ -909,8 +909,6 @@ impl VMBuiltinFunctionsArray {
 /// The struct here is empty, as the sizes of these fields are dynamic, and
 /// we can't describe them in Rust's type system. Sufficient memory is
 /// allocated at runtime.
-///
-/// TODO: We could move the globals into the `vmctx` allocation too.
```

Comment:
Looking at `InstanceHandle`, it looks like globals have an extra layer of indirection on them

```Rust
            instance.globals_ptr() as *mut NonNull<VMGlobalDefinition>,
```

Whereas everything else only has 1, so it looks like this comment is still valid.

We could probably fix this in the same way that #1687 is fixing memory/table